app/src/lib/helpers/linux.ts

```diff
@@ -84,7 +84,7 @@ export function spawnEditor(
   if (isFlatpakBuild()) {
     return spawn(
       'flatpak-spawn',
-      ['--host', `"${path}"`, `"${workingDirectory}"`],
+      ['--host', path, `"${workingDirectory}"`],
```

Comment:
We originally had some issues with unescaped paths not being handled by editors/shells on other OSes - e.g. space in path turning one parameter into two - which is why `workingDirectory` is quoted like this. Not sure why `path` would be partially quoted but given we know the subset of paths that the app will look for, this feels like a low-risk change 👍🏻

Reply:
I'm not worried about the path to the binary being escaped because at least on linux we know there won't be anything to escape.  the partially quoted issue comes from `workingDirectory`. I've only been able to reproduce it a handful of times. with `path` in quotes for some reason the editor would never launch

Reply:
Let's see if we can cut a release to simplify your Flatpak build and maybe that will yield the problem behaviour you were seeing ...